docs/resources/faq.md

```diff
@@ -0,0 +1,187 @@
+# Frequently Asked Questions
+Various resources, including [Facebook](https://www.facebook.com/groups/diytrio), and [Discord](https://discord.com/invite/FnwFEFUwXE), are available to assist you with setting up, maintaining, and succeeding in your Trio journey. Below is a series of questions we see asked daily. We hope it helps. 
+
+## Trio FAQs
+### What is Trio?
+Trio is an open-source artificial pancreas system based on the OpenAPS (Oref0) algorithm. It uses your entered settings, carbohydrates, and historical data to automate insulin delivery and reduce the time spent managing your diabetes. 
+ 
+The project started with Ivan Valkou's FreeAPS X implementation of the OpenAPS algorithm for iPhone and was later forked and rebranded as iAPS. Since then, many developers have contributed substantial work, leading to new features and enhancements.
+
+Following the release of version 3.0.0, the project’s direction significantly changed due to differing views on development, open source, and peer review. This led to the separation from the iAPS repository and the birth of Trio as a distinct entity. This transition marks the project’s new phase, symbolizing its evolution and the collaborative development’s dynamic nature.
+
+!!! important "Important"
+_Trio is a DIY, open-source community project. As such, the user must build and maintain their app with the support of the Trio community. Trio has not undergone evaluation by any regulatory agency and is for use at your own risk._ 
```

Comment:
In this file, each line of the **body text** of each admonition, must be indented 4-spaces, like this:

```markdown
!!! important "Important"
    _Trio is a DIY, open-source community project. As such, the user must build and maintain their app with the support of the Trio community. Trio has not undergone evaluation by any regulatory agency and is for use at your own risk._ 
```

It currently looks like this:

> ![CleanShot 2024-11-13 at 21 32 19@2x](https://github.com/user-attachments/assets/52a23923-9f9c-46d8-8a85-2b3cc59c615e)

whereas it should look like so:

> ![CleanShot 2024-11-13 at 21 33 38@2x](https://github.com/user-attachments/assets/8f833412-e9ee-41ca-a4c8-e6f2dcd64ac7)


Source: https://github.com/nightscout/trio-docs/tree/082d0729fe32ba22e622ad0b7abdd2f97917fa94#migrate-an-admonition-from-sphinx-to-mkdocs

Reply:
Thank you very much; I kept making changes to troubleshoot why it wasn't displaying correctly.

I have gone through and made the changes as per the link provided. However, when I build locally, the admonition still displays incorrectly. Has the issue been fixed on your end?